src/elements/content-sidebar/messages.js

```diff
@@ -117,6 +117,11 @@ const messages = defineMessages({
         defaultMessage: 'Modify General Task',
         description: 'modal title for when editing an existing general task',
     },
+    expandBoxAI: {
+        id: 'be.expandBoxAI',
```

Comment:
we should add `be.contentSidebar` to the id prefix since we dont want to risk any conflicts